hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/test/java/org/apache/hadoop/yarn/server/router/rmadmin/TestableFederationRMAdminInterceptor.java

```diff
@@ -46,7 +48,6 @@ public class TestableFederationRMAdminInterceptor extends FederationRMAdminInter
 
   // Used to Store the relationship between SubClusterId and RM
   private ConcurrentHashMap<SubClusterId, MockRM> mockRMs = new ConcurrentHashMap<>();
-
```

Comment:
I would leave this as it was.

Reply:
Thank you very much for your help to review the code, I will modify the code.